.github/workflows/ci.yaml

```diff
@@ -70,7 +70,7 @@ jobs:
       - uses: actions-rs/toolchain@v1
         with:
           profile: minimal
-          toolchain: nightly-2022-11-01
+          toolchain: 1.65.0
           override: true
```

Comment:
I assume we can just omit this line and `toolchain` now, since it matches the format file.

Reply:
(But, merging anyway, I don't mind if it's explicit. Welcome to change in a follow-up commit.)

Reply:
Ah yes according to the [action documentation](https://github.com/actions-rs/toolchain#the-toolchain-file) that should work.